src/components/ExpandedNode/ExpandedNode.tsx

```diff
@@ -383,17 +384,32 @@ const ExpandedNodeMemo: FC<ExpandedNodeMemoProps> = memo(
     isAccepted,
     onAdvance
   }) => {
+    const { patientRecords } = usePatientRecords();
     const guidance = isGuidance && renderGuidance(pathwayState, documentation, isAccepted);
     const branch =
       isBranchState(pathwayState) && renderBranch(documentation, pathwayState, isAccepted);
     const defaultText =
       'The patient and I discussed the treatment plan, risks, benefits and alternatives.  The patient expressed understanding and wants to proceed.';
+
+    let notes;
+    const documentReference = retrieveNote(pathwayState.label, patientRecords);
+    if (documentReference) {
+      const content = documentReference.content[0].attachment?.data;
+      if (content) {
+        notes = atob(content);
+        notes = notes.slice(notes.indexOf('Notes: ') + 7)
+      }
+    }
+
     return (
       <div className={indexStyles.expandedNode}>
         <table className={styles.infoTable}>
           <tbody>
             <StatusField documentation={documentation} isAccepted={isAccepted} />
             {guidance || branch}
+            {!isActionable && notes && /\S/.test(notes) && (
```

Comment:
Do we still need what I presume is the whitespace check here?
```suggestion
            {!isActionable && notes && (
```

Reply:
I think so, in case someone tries to submit a comment that has nothing but a whitespace (i.e. accidentally pressed the spacebar or return) then this catches that and stops the comments title from being visible with a blank description box.

Reply:
Good enough to convince me

Reply:
<img width="445" alt="Screen Shot 2020-06-09 at 2 52 35 PM" src="https://user-images.githubusercontent.com/37051655/84188032-ed1e7500-aa60-11ea-8f68-47c64f202b6e.png">